File: src/main/modules/microarray/R-2.12/input-output.R
# TOOL input-output.R: InputOutput (Empty analysis for testing system wide data transfer.)
# INPUT input.tsv: input.tsv TYPE GENERIC 
# OUTPUT output.tsv: output.tsv 

#input<-read.table(c("input.tsv"), header=T, sep="\t")
#system("mv input.tsv output.tsv")
write.table(c(1, 2, 3), file="output.tsv", quote=FALSE, col.names=FALSE, row.names=FALSE)

File: src/main/modules/microarray/python/input-output.py
# TOOL input-output.py: InputOutputPython (Empty analysis for testing Python support..)
# INPUT input.tsv: input.tsv TYPE GENERIC 

print 'Works!'


# OUTPUT output.tsv: output.tsv 
import csv

with open('input.tsv', 'rb') as infile:
	reader = csv.reader(infile, delimiter='\t')
	with open('output.tsv', 'wb') as outfile:
		writer = csv.writer(outfile, delimiter='\t')
		for row in reader:
			writer.writerow(row)